docs/pages/guides/Importing-Apps.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: default
+title: Importing  Applications
+description: This guide is to help users get a better understanding of sample apps
+permalink: guides/importing-apps
+type: Guides
+language: en
+---
+
+
+Importing apps into Meshery as Meshery Apps is a powerful feature that enables users to manage, operate, and observe their service mesh more effectively. With this feature, users can easily import their app manifest and store it in the database.
+
+You can import apps into Meshery using either the Meshery CLI or the Meshery UI. We will discuss both methods in detail below.
+
+
+## Using Meshery CLI:
+
+
+**Step 1: Install Meshery CLI**
+
+Before you can use the Meshery CLI to import a Docker Compose app, you must first install it. You can install Meshery CLI by following the instructions on the Meshery documentation site.
+
+
+**Step 2: Import the App Manifest**
+
+Once you have created your App Definition file, you can use the Meshery CLI to import your Docker Compose app into Meshery. To do this, run the following command:
+
+
+
+<pre class="codeblock-pre">
+<div class="codeblock"><div class="clipboardjs">mesheryctl app import -f [file/url] -s [source-type]</div></div>
+</pre>
+
+This command enable users to import their existing applications from 
+- Helm Charts
+- k8s manifests
+- Docker Compose
```

Comment:
It'd be great if an example command is present

Reply:
done!